content/team/index.md

```diff
@@ -0,0 +1,258 @@
+---
+pagetype: major
+pagename: team
+section: Team
+title: Team | IPFS
+url: team
+save_as: team/index.html
+constellation: constellation-02.svg
+---
+
+The IPFS Team is a collection of thousands of contributors from multiple geographies and backgrounds, interested in making the Web Distributed.
+
+```
+TODO Insert nice animation with World Map and hex faces
```

Comment:
We had feedback around this when we talked about improving the PL site last summer. The feedback was: that animation was hard to keep up to date, and didn't properly reflect the community. We shouldn't use it again here.